src/services/data-exports/export-cnfs/repositories/export-cnfs.repository.js

```diff
@@ -38,14 +40,27 @@ const getStatsCnfs = db => async (dateDebut, dateFin, nomOrdre, ordre, certifie,
     emailCNError: 1,
     mattermost: 1,
   });
+  const arrayConseillers = [];
 
   if (nomOrdre !== undefined && ordre !== undefined) {
-    return conseillers
+    const ordreResult = conseillers
     .sort({ [nomOrdre]: parseInt(ordre) })
     .toArray();
+    for (let conseiller of await ordreResult) {
+      const result = await getCraCount(db)(conseiller);
+      conseiller.craCount = result;
+      arrayConseillers.push(conseiller);
+    }
+    return await arrayConseillers;
```

Comment:
normal le await ici ?

Reply:
alors effectivement il faut que j'utilise le await sois ici ou alors dans le for ! ça fais doublon, je l'enlève...